eng/testing/tests.wasm.targets

```diff
@@ -16,7 +16,7 @@
     <_ShellCommandSeparator Condition="'$(OS)' == 'Windows_NT'">&amp;&amp;</_ShellCommandSeparator>
     <_ShellCommandSeparator Condition="'$(OS)' != 'Windows_NT'">&amp;&amp;</_ShellCommandSeparator>
     <XUnitUseRandomizedTestOrderer Condition="'$(XUnitUseRandomizedTestOrderer)' == '' and '$(IsTestProject)' == 'true'">true</XUnitUseRandomizedTestOrderer>
-    <_XHarnessTestsTimeout>00:30:00</_XHarnessTestsTimeout>
+    <_XHarnessTestsTimeout Condition="'$(_XHarnessTestsTimeout)' == ''">00:30:00</_XHarnessTestsTimeout>
```

Comment:
Rename this to not be private like `WasmXHarnessTestsTimeout`, so we are not using a private property in a different project.